src/cli/test/test_cli.c

```diff
@@ -1938,8 +1938,6 @@ static void test_trn_cli_update_packet_metadata_egress_bw_subcmd(void **state)
 	struct rpc_trn_packet_metadata_t exp_packet_metadata = {
 		.interface = "eth0",
 		.ip = 0x100000a,
-		.pod_label_value = 0,
-		.namespace_label_value = 0,
```

Comment:
Minor. I'm a little confused that here the label values are deleted (which is ok) but later in line 2107 the values are added. I think we can either need them in those places or remove them in those places.

Reply:
This is intentional, I want the tests to exercise cases where the labels are missing and cases where they are present to ensure that my changes can co-exist with the label changes.